src/request.js

```diff
@@ -137,6 +138,18 @@ export function buildRetries(context) {
     return DEFAULT_RETRIES;
 }
 
+export function buildProxyRetries(context) {
+    const contextProxyRetries = get(context, 'options.proxyRetries');
+    if (contextProxyRetries) {
```

Comment:
just so im understanding this correctly- the app can configure this, or we can get it from an env variable, or we go with a default set here- which is going to be 0?

Reply:
This how we enable this for Styx on all services: https://github.com/globality-corp/ansible-deploy/pull/4003